.github/workflows/ci.yml

```diff
@@ -72,6 +72,7 @@ jobs:
         cmake -H"." -B"build"        \
           -DCMAKE_BUILD_TYPE="Debug" \
           -DBUILD_C=ON               \
+          -DENABLE_MPI=ON            \
```

Comment:
if the macro is not from cmake I would suggest adding cosimio prefix to avoid conflicts with host macros.

Reply:
done